.github/workflows/publish/publish.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+alias="s3"
+
+mc config host add "${alias}" "https://s3.amazonaws.com" "${AWS_ACCESS_KEY}" "${AWS_SECRET_KEY}"
+mc cp output/scf-*.tgz "${alias}/${BUCKET}"
+echo "URL: https://${BUCKET}.s3.amazonaws.com/$(ls output/scf-*.tgz)"
```

Comment:
Huh, the file is `https://bucket.s3.amazonaws.com/output/scf-foo.tgz`? (I think the `ls` on the last line will show the `output/` directory name too, which seems incorrect.)

Reply:
Oh, you are right.